packages/manager/apps/hub-next/src/main.ts

```diff
@@ -0,0 +1,44 @@
+import { createApp } from 'vue';
+import { createI18n } from 'vue-i18n';
+import { attach } from '@ovh-ux/manager-preloader';
+import { fetchConfiguration, Environment, LANGUAGES } from '@ovh-ux/manager-config';
+import { loadLocaleMessages } from '@/i18n';
+
+// css
+import 'bootstrap/dist/css/bootstrap.min.css';
+import '@ovh-ux/ui-kit/dist/css/oui.css';
+
+import App from './App.vue';
+import router from './router';
+import store from './store';
+
+fetchConfiguration('hub').then(async () => {
+  const locale = Environment.getUserLocale();
+  const fallbackLocale = LANGUAGES.fallback;
+  const datetimeFormats = {
+    [locale.replace('_', '-')]: {
+      short: { year: 'numeric', month: 'short', day: 'numeric' },
+      shortNumeric: { year: 'numeric', month: 'numeric', day: 'numeric' },
+    },
+  };
+  attach(Environment.getUserLanguage());
+
+  const i18n = createI18n({
+    locale,
+    fallbackLocale,
+    datetimeFormats,
+  });
+
+  router.beforeEach(async (to, from, next) => {
+    const translationFolders = to.meta.relatedTranslations;
```

Comment:
If I understand well, here you use the Router to load translations files. Should it be possible to use the same way as we do now in manager? I mean loading asynchronously translations with their corresponding components? Like so, translations are not linked to a route but to a component.

Reply:
Theoretically, it should be possible though i never tried it. I'll try it and see how it goes. It could be better tbh
I'll keep u updated

Reply:
Cool! 
In fact, this implementation is the way we did before. It's not a bad solution but, load the translations asynchronously allowed us to re-use our components easily.